docker-images/preinstall-hook/etcdStatus.sh

```diff
@@ -26,10 +26,17 @@ echo "Verifying if the provided etcd url is accessible: $etcdURL"
 #Verify with certs if it is a secured etcd. 
 if [[ ! -z $etcdca ]]
 then
-    echo "Verifying connectivity to secure etcd... The certs need to be at the location $etcdca $etcdcert and $etcdkey"
-    response=$(curl --write-out %{http_code} --silent --output /dev/null --cacert $etcdca --cert $etcdcert --key $etcdkey -L "$etcdURL/version" )
-    echo "Response Code: $response"
-    verifyresponse $response
+    if [[ ! -z $etcdcert ]]
```

Comment:
We recently moved the source for the preinstall-hook at https://github.com/portworx/px-preinstall-hook. I'll port your changes there.

Reply:
https://github.com/portworx/px-preinstall-hook/pull/2 for the port

@hrishipx can you help merge that?